.github/workflows/python.yml

```diff
@@ -2,7 +2,10 @@ name: Python CI/CD
 
 on:
   push:
+    branches: [ '**' ]
+    tags-ignore: [ '**' ]
```

Comment:
What is this doing?

Reply:
This is just a cleanup for releases. A release triggers two different CI runs: release and tag. Since we upload to PyPI using the publish event associated to the release, running also on tags is disabled.